### PR TITLE
Create 2021-12-07-FORCE2021-lightening.md

### LINKED ISSUE #2098: Submitting abstract for FORCE 2021
Hi all, 

I would like to **submit an abstract for the upcoming FORCE 2021 conference** (7-9 December): https://www.force11.org/meetings/force2021

The deadline for submission of lightening talks is **15th of October**: https://docs.google.com/forms/d/e/1FAIpQLScU6To6FzElmBMdnJkf8E18fGdID7MQMhFIVmHqrN59_tbV8g/viewform

**Example of past talks** can be found here: https://zenodo.org/communities/force2019/?page=1&size=20 (thanks @EKaroune!)

**Speaker:** Esther Plomp, unless someone else would like to volunteer!  (Emma has other commitments)

**Draft text**, adapted from CarpentryCon at home: https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/2020-08-CarpentryCon-at-home-lightning.md (see also the PR!)

**Title:** The Turing Way: A sustainable and inclusive solution to sharing best practices.

The Turing Way is a community-driven book project that involves and supports a diverse research community in ensuring that reproducible and ethical data research is "too easy not to do". The book is openly available under a CC-BY 4.0 licence, and presents a sustainable alternative to the more traditional way of sharing best practices via articles. 

The Turing Way aims to bridge the gap between innovative data research techniques and best practices that make them accessible and comprehensible for everyone. The Turing Way is designed to be inclusive and accessible to its members from diverse backgrounds and expertise. We welcome ideas, case studies, impact stories, educational resources, or any sort of participation that individuals can benefit from or use to advance their knowledge in data research. All contributions, questions, comments, and discussions via an online repository: https://github.com/alan-turing-institute/the-turing-way. This ensures that all the contributions are openly accessible. To ensure long term preservation a snapshot of the repository is also hosted on a data repository (10.5281/zenodo.3233853).

In this lightning talk, a community member will introduce the Turing Way Project and highlight the importance of creating a culture that promotes collaboration and accessibility to ensure the long-term sustainability of our communities.

**What else needs to be done?**
@malvikasharan: I would like to highlight to this particular community that we have a version of the Turing Way archived at Zenodo, but it seems that the last update was from 2019? Would it be possible to update this before the conference in December?

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Fixes #2098

### List of changes proposed in this PR (pull-request)

* Add abstract for abstract submission to FORCE2021

### What should a reviewer concentrate their feedback on?

- [ ] Does the abstract sound like its about scholarly communication?
- [ ] Everything looks ok?
- [ ] Would you like to present?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: conferences/abstracts/2021-12-07-FORCE2021-lightening.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+*Please supply one paragraph describing your presentation and how it relates to the tracks above and the theme of The Future of Scholarly Communication.*
+
+The Turing Way is a community-driven book project that involves and supports a diverse research community in ensuring that reproducible and ethical data research is "too easy not to do". 
+The book is openly available under a CC-BY 4.0 licence, and presents a sustainable alternative to the more traditional way of sharing best practices via articles.
+
+The Turing Way aims to bridge the gap between innovative data research techniques and best practices that make them accessible and comprehensible for everyone. 
+The Turing Way is designed to be inclusive and accessible to its members from diverse backgrounds and expertise. 
+We welcome ideas, case studies, impact stories, educational resources, or any sort of participation that individuals can benefit from or use to advance their knowledge in data research. 
+All contributions, questions, comments, and discussions via an online repository: https://github.com/alan-turing-institute/the-turing-way.
```
Comment (EKaroune) [inline conferences/abstracts/2021-12-07-FORCE2021-lightening.md:32]:
```suggestion
All contributions, questions, comments, and discussions are made via an online repository: https://github.com/alan-turing-institute/the-turing-way. 
```

File: conferences/abstracts/2021-12-07-FORCE2021-lightening.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+The Turing Way is a community-driven book project that involves and supports a diverse research community in ensuring that reproducible and ethical data research is "too easy not to do". 
+The book is openly available under a CC-BY 4.0 licence, and presents a sustainable alternative to the more traditional way of sharing best practices via articles.
+
+The Turing Way aims to bridge the gap between innovative data research techniques and best practices that make them accessible and comprehensible for everyone. 
+The Turing Way is designed to be inclusive and accessible to its members from diverse backgrounds and expertise. 
+We welcome ideas, case studies, impact stories, educational resources, or any sort of participation that individuals can benefit from or use to advance their knowledge in data research. 
+All contributions, questions, comments, and discussions via an online repository: https://github.com/alan-turing-institute/the-turing-way. 
+This ensures that all the contributions are openly accessible. 
+To ensure long term preservation a snapshot of the repository is also hosted on a data repository (10.5281/zenodo.3233853).
```
Comment (EKaroune) [inline conferences/abstracts/2021-12-07-FORCE2021-lightening.md:34]:
```suggestion
To ensure long term preservation, a snapshot of the repository is also hosted on a data repository (10.5281/zenodo.3233853).
```

**Review by EKaroune — APPROVED** [review]
This looks great! 😄

**Comment by malvikasharan** [comment]
@EstherPlomp it is possible. I only need accountability - the draft version for release has been sitting for over 4 months :(

**Comment by EstherPlomp** [comment]
> 
> 
> @EstherPlomp it is possible. I only need accountability - the draft version for release has been sitting for over 4 months :( Can we work on it next coworking call?

I'm not going to be able to make the co working calls on the 4th and 11th due to a training and the dentist/holidays :( Happy to work on this on the 18th of October if that also works for you!

**Pull request merged**